### ISSUE TITLE
PIllView is never remove from the view hierarchy after being dismissed

### ISSUE THREAD

**Issue by martindufort** [Awesomeplayer165/PillboxView#25]
When showing a PillView, the view is added to the `vcView` hierarchy. 
However when dismissing the `PillView` all internals elements are removed but the `PillView` is never removed from the hierarchy. 

See these lines: 
https://github.com/Awesomeplayer165/PillboxView/blob/d5f7cbfed3e914bc051ca275ede3f9dfa7c367c1/Sources/PillboxView/PillView.swift#L258

https://github.com/Awesomeplayer165/PillboxView/blob/d5f7cbfed3e914bc051ca275ede3f9dfa7c367c1/Sources/PillboxView/PillView.swift#L171

https://github.com/Awesomeplayer165/PillboxView/blob/d5f7cbfed3e914bc051ca275ede3f9dfa7c367c1/Sources/PillboxView/PillView.swift#L326

https://github.com/Awesomeplayer165/PillboxView/blob/d5f7cbfed3e914bc051ca275ede3f9dfa7c367c1/Sources/PillboxView/PillView.swift#L333

**Expected behavior**
The `PillView` should be removed from the `vcView` hierarchy after being dismissed

**Comment by Awesomeplayer165** [comment]
You're right! Thanks for pointing this out. I'll look into this more later.
